### PR TITLE
test(facet actions): facet actions test

### PR DESCRIPTION
**Pull request by deepunbxd005**

### CODE REVIEW

File: src/modules/textFacets/__tests__/textFacets.test.js [hunk 1]
```diff
@@ -100,6 +114,158 @@ test('Match Snapshot for text facets', async () => {
...
+
+    await waitFor(() => {
+        expect(getByText('Scarpa - 18')).toBeInTheDocument();
+        fireEvent.click(getByText("Scarpa - 18"));
+        fireEvent.click(getByText("Apply Facets"));
+        expect(onApply).toHaveBeenCalledTimes(1);
+    })
+});
+
+test('Test text facet actions with custom onApply method', async () => {
```
Comment (vikramgupta1988) [inline src/modules/textFacets/__tests__/textFacets.test.js:242]: custom onCLear

File: src/modules/textFacets/__tests__/textFacets.test.js [hunk 2]
```diff
@@ -100,6 +114,158 @@ test('Match Snapshot for text facets', async () => {
...
+});
+
+test('Test text facet actions showClearFilter', async () => {
+    const { container } = render(
+        <>
+            <UnbxdSearchWrapper
+                siteKey="wildearthclone-neto-com-au808941566310465"
+                apiKey="e6959ae0b643d51b565dc3e01bf41ec1"
+            >   
+                <FacetActions showClearFilter={false} />
```
Comment (vikramgupta1988) [inline src/modules/textFacets/__tests__/textFacets.test.js:173]: By default, is it true?

**Review by vikramgupta1988 — CHANGES_REQUESTED** [review]

File: src/modules/textFacets/__tests__/textFacets.test.js (hunk 2 above)
Comment (deepunbxd005) [reply src/modules/textFacets/__tests__/textFacets.test.js:173]: yes

Comment (deepunbxd005) [reply src/modules/textFacets/__tests__/textFacets.test.js:173]: as we can see in the snapshot created

File: src/modules/textFacets/__tests__/textFacets.test.js (hunk 1 above)
Comment (deepunbxd005) [reply src/modules/textFacets/__tests__/textFacets.test.js]: done

**Review by vikramgupta1988 — APPROVED** [review]

**Pull request merged**